pkg/router/base_rule.go

```diff
@@ -33,6 +37,10 @@ import (
 	"mosn.io/mosn/pkg/upstream/cluster"
 )
 
+var (
+	schemeValidator = regexp.MustCompile("^[a-z][a-z0-9.+-]*$")
```

Comment:
You can annotate the RFC referenced below.

Reply:
added